### PR TITLE
Update legacy theme

### LINKED ISSUE #1466: Update legacy theme
# Issue Description
The terra-legacy-theme package has become outdated with our theming approach. We should update it to match our approach of theming with CSS custom properties

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
terra-legacy-theme is updated to use CSS custom properties for theming

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #1466 

### Deployment URL
https://terra-core-deployed-pr-1467.herokuapp.com/

To view the legacy theme, navigate to the deployment URL, select the legacy theme in the config dropdown in the top right.

### CODE REVIEW

File: packages/terra-legacy-theme/README.md [hunk 1]
```diff
@@ -3,29 +3,112 @@
...
+In your application, import the `terra-legacy-theme` module like so:
+
+```js
+// ES6 import
+import 'terra-legacy-theme';
+```
+
+#### Format 1 Setup
+
+You'll need to set up some PostCSS tools to ensure the terra-legacy-theme can be compiled correctly.
```
Comment (JakeLaCombe) [inline packages/terra-legacy-theme/README.md]: How much of this is covered by Terra toolkit? It a lot of it is, it maybe better to specify using terra-toolkit?

Comment (bjankord) [reply packages/terra-legacy-theme/README.md]: Added a note about using terra-toolkit: https://github.com/cerner/terra-core/pull/1467/commits/c7460970d56bb0b7176e3509cdc5dd37714c4ccf

File: packages/terra-legacy-theme/README.md [hunk 2]
```diff
@@ -3,29 +3,114 @@
...
+
+* [postcss-assets-webpack-plugin](https://github.com/klimashkin/postcss-assets-webpack-plugin)
+* [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties)
+
+`npm install postcss-assets-webpack-plugin --save-dev`
+
+`npm install postcss-custom-properties --save-dev`
+
+The postcss-assets-webpack-plugin gets css, extracted by [ExtractTextPlugin](https://github.com/webpack/extract-text-webpack-plugin) and applies postcss plugins to it.
```
Comment (yuderekyu) [inline packages/terra-legacy-theme/README.md]: We should point out that `extract-text-webpack-plugin` doesn't have webpack 4 support. If consumers use their own webpack config utilizing webpack v4, they should use `mini-css-extract-plugin`.

**Comment by mmalaker** [comment]
Functional verification completed. The Terra site navigation displays and functions correctly with the legacy theme applied.

**Pull request merged**